tests/basic/pool_function_test.go

```diff
@@ -388,3 +389,103 @@ var _ = Describe("{PoolExpansionDiskResizeInvalidSize}", func() {
 	})
 
 })
+
+var _ = Describe("{PoolExpandResizeWithSameSize}", func() {
+
+	BeforeEach(func() {
+		StartTorpedoTest("PoolExpandResizeWithSameSize",
+			"Initiate pool expansion using same size", nil, testrailID)
+	})
+
```

Comment:
As a general practice, lets deploy apps before start of each test and destroy them after test finishes.

Reply:
yes,